src/index.css

```diff
@@ -0,0 +1,71 @@
+:root {
```

Comment:
- [ ] In the desktop view, there is some inconsistency with the navigation, and the colors:

Add doctor has a green hamburger menu:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/11c1d034-c398-4096-8c79-96eda296f4bf)

Add doctor has a green arrow icon:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/fce1adf9-0e4a-4631-8e17-1491ba86ae89)

Delete Doctor has an arrow button, and it is blue:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/4d7d2ef5-9194-4045-8392-f956c7b4b2e5)


Kindly, make it so the means of navigation is more consistent, and all the arrows/hamburgers have the same color to match the design. 👍